Index: google-breakpad/src/processor/minidump_stackwalk.cc
===================================================================
--- google-breakpad/src/processor/minidump_stackwalk.cc	(revision 319)
+++ google-breakpad/src/processor/minidump_stackwalk.cc	(working copy)
@@ -34,6 +34,7 @@
 
 #include <cstdio>
 #include <cstdlib>
+#include <cstring>
 #include <string>
 #include <vector>
 
Index: google-breakpad/src/tools/linux/symupload/sym_upload.cc
===================================================================
--- google-breakpad/src/tools/linux/symupload/sym_upload.cc	(revision 319)
+++ google-breakpad/src/tools/linux/symupload/sym_upload.cc	(working copy)
@@ -41,6 +41,7 @@
 
 #include <unistd.h>
 
+#include <cstdlib>
 #include <cstdio>
 #include <cassert>
 #include <functional>
Index: google-breakpad/src/tools/linux/symupload/minidump_upload.cc
===================================================================
--- google-breakpad/src/tools/linux/symupload/minidump_upload.cc	(revision 319)
+++ google-breakpad/src/tools/linux/symupload/minidump_upload.cc	(working copy)
@@ -34,6 +34,7 @@
 //  symbol_file: the breakpad format symbol file
 
 #include <unistd.h>
+#include <cstdlib>
 
 #include <string>
 
Index: google-breakpad/src/client/linux/handler/exception_handler.cc
===================================================================
--- google-breakpad/src/client/linux/handler/exception_handler.cc	(revision 319)
+++ google-breakpad/src/client/linux/handler/exception_handler.cc	(working copy)
@@ -37,6 +37,7 @@
 #include <cassert>
 #include <cstdlib>
 #include <ctime>
+#include <cstring>
 #include <linux/limits.h>
 
 #include "client/linux/handler/exception_handler.h"
